pkg/sql/conn_executor.go

```diff
@@ -937,6 +939,11 @@ type connExecutor struct {
 		// that staged them commits.
 		jobs jobsCollection
 
+		// jobsCache is a map of TableIDs to Jobs.
```

Comment:
Can you change this comment to be just descriptor IDs? We're going to have jobs tracking changes to descriptors other than tables in the future.